package.json

```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "arrify": "^1.0.1",
     "chalk": "^1.1.3",
+    "minimatch": "^3.0.3",
```

Comment:
@sergesemashko just curious, is the filter fn that you're using from minimatch available from `node-glob`? could remove unnecessary dependencies :)